Library/Formula/bish.rb

```diff
@@ -0,0 +1,27 @@
+class Bish < Formula
+  homepage "https://github.com/tdenniston/bish"
+  url "https://github.com/tdenniston/bish/archive/v0.1.tar.gz"
+  sha256 "796d1efcbc9e8c7ea604881860aa8d857aad089c4eeb766283c21c210687942b"
+
+  def install
+    stdlib_path = share/"stdlib.bish"
+    mkdir share
+    mv "lib/stdlib.bish", stdlib_path
```

Comment:
Thanks! Yeah - It essentially does the same thing, but it looks less messy in formulae which is always a :+1:.